src/tactic/core.lean

```diff
@@ -1805,6 +1817,7 @@ do new_decl_type ← declaration.type <$> get_decl new_decl_name,
    (_, inst) ← solve_aux tgt
      (intros >> reset_instance_cache >> delta_target [new_decl_name]  >> apply_instance >> done),
    inst ← instantiate_mvars inst,
+   inst ← replace_univ_metas_with_univ_params inst,
```

Comment:
Do you know where the universe metavariables come from here?

Reply:
In my examples that provoked this PR, they were the morphism universe levels associated to a category instance. In particular, see the test case I added:

```lean
attribute [derive large_category] Mon
```

which used to fail with a "declaration has universe metavariables" type error.